change/@microsoft-fast-foundation-96d671a6-621c-4790-8217-b319bdc883d3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "tree view events",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "scomea@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
Should this be *minor* since, while it fixes a broken behavior, still introduces a new behavior?

Reply:
@scomea this change should be all that's needed IMO